server/config/routes.rb

```diff
@@ -16,6 +16,13 @@
         resources :payment_groups, only: %i[index show], module: :management_groups do
           resources :payment_affiliations, only: %i[index], module: :payment_groups
         end
+        scope module: :management_groups do
+          namespace :payment_groups do
+            resources :payment_affiliations, only: %i[] do
+              post 'bulk_insert', on: :collection
+            end
```

Comment:
```suggestion
            namespace :payment_affiliations do
              post 'bulk_insert'
            end
```

とかだと無理そうなのか。action作らないのにわざわざresources使わなくていい気がした。

Reply:
厳しそう
```
api_v1_management_group_payment_groups_payment_affiliations_bulk_insert POST   /api/v1/management_groups/:management_group_id/payment_groups/payment_affiliations/bulk_insert(.:format)       api/v1/management_groups/payment_groups/payment_affiliations/management_groups#bulk_insert
```

これは動きそう
```ruby
scope module: :management_groups do
  namespace :payment_groups do
    scope :payment_affiliations do
      post :bulk_insert, to: 'payment_affiliations#bulk_insert'
    end
  end
end
```